CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ---
 
+## [0.6.1] - 2020-08-04
```

Comment:
To enable the link on this title, you also need to update the URLs in the bottom of the file...

https://github.com/global-121/121-platform/pull/1068/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR83-R84